src/Merchants/Content/Merchant/MerchantEntity.php

```diff
@@ -3,35 +3,179 @@
 namespace Shopware\Production\Merchants\Content\Merchant;
 
 use Shopware\Core\Checkout\Customer\CustomerCollection;
+use Shopware\Core\Checkout\Customer\CustomerEntity;
+use Shopware\Core\Content\Category\CategoryEntity;
+use Shopware\Core\Content\Product\ProductCollection;
 use Shopware\Core\Framework\DataAbstractionLayer\Entity;
 use Shopware\Core\Framework\DataAbstractionLayer\EntityIdTrait;
+use Shopware\Core\System\SalesChannel\SalesChannelEntity;
 
 class MerchantEntity extends Entity
 {
     use EntityIdTrait;
 
     /**
-     * @var null|string
+     * @var string|null
      */
-    public $customerId;
+    protected $name;
 
     /**
-     * @var null|string
+     * @var string|null
      */
-    public $name;
+    protected $website;
 
     /**
-     * @var null|string
+     * @var string|null
      */
-    public $website;
+    protected $description;
 
     /**
-     * @var null|string
+     * @var string|null
      */
-    public $phoneNumber;
+    protected $phoneNumber;
 
     /**
-     * @var null|CustomerCollection
+     * @var string|null
      */
-    protected $customers;
+    protected $customerId;
+
+    /**
+     * @var CustomerEntity|null
+     */
+    protected $customer;
+
+    /**
+     * @var string|null
+     */
+    protected $salesChannelId;
+
+    /**
+     * @var SalesChannelEntity|null
+     */
+    protected $salesChannel;
+
+    /**
+     * @var string|null
+     */
+    protected $categoryId;
+
+    /**
+     * @var CategoryEntity
```

Comment:
`CategoryEntity|null` ? Also for the typehints. Any reason why this is not null?